src/Accordion/Accordion.tsx

```diff
@@ -27,35 +27,42 @@ export interface AccordionProps {
     iconRotation?: AccordionIconRotation;
     id?: string;
     list: Array<AccrodionListItem>;
+    alternative?: boolean;
 }
 
 const Accordion: React.FunctionComponent<AccordionProps> = (props: AccordionProps) => {
     const [active, setActive] = React.useState<number>(null);
-    const [className, setClassName] = React.useState<string>("custom-accordion");
+    const [itemClassName, setItemClassName] = React.useState<string>("custom-accordion");
     const [idList, setIdList] = React.useState<Array<string>>([]);
 
     React.useEffect(() => {
         constructIds();
-        constructClassName();
+        constructItemClassName();
     }, []);
 
     React.useEffect(() => {
         constructIds();
-        constructClassName();
+        constructItemClassName();
     }, [props.id, props.className, props.iconPosition, props.iconRotation, props.customIconExpanded, props.list]);
 
+    /**
+     * Constructs the component's `id` if `id` prop is passed
+     */
     function constructIds(): void {
         const idListToSet: Array<string> = [];
         props.list.map(() => idListToSet.push(randomId("accordion-")));
         setIdList(idListToSet);
     }
 
-    function constructClassName(): void {
+    /**
+     * Constructs the `classname` to be used in accordion items
+     */
+    function constructItemClassName(): void {
         let cn: string = "accordion-item";
```

Comment:
I suggest to add this as a utility, this function can help set className when the criterias are too much to track especially for handling spaces:
```
function classList(classes) {
  return Object
    .entries(classes)
    .filter(entry => entry[1])
    .map(entry => entry[0])
    .join(' ');
}

```
The logic is easy to understand:

1. First, it transforms our input object in an array of key-value pairs (one pair for each property).
2. Then, it filters the resulting array to eliminate items whose toggling criteria evaluate to false.
3. After that, it maps each item of the filtered array to its respective class name, outputting a new array with a list of classes.
4. Last, but not least, it joins all classes using a space as separator. The function returns the resulting string.

Running this will return => "accordion-item otherClass"
```
classNames({
  'accordion-item': true,
  'otherClass': true,
  'probablyNotClass': false,
})
```

Reply:
new state is introduced to cater for multiple classes in component